README.md

```diff
@@ -70,6 +75,26 @@ If you need to install the library globally
 sudo make install
 ```
 
+We could also use Microsoft's C++ Library Manager 
+[`vcpkg`](https://github.com/Microsoft/vcpkg) to install Benchmark in our systems. 
+This manager works in Windows, Linux and Mac OS and it is very easy to manage 
+libraries with it. To install Benchmark we simply need to run the following command, 
+provided we already have `vcpkg` installed in our system with user-wide integration. 
+(NOTE: documentation for installation of `vcpkg` can be found 
+[here](https://github.com/Microsoft/vcpkg))
+
+For Windows
+```bash
+PS> .\vcpkg install benchmark
+```
+For Linux
+```bash
+$ ./vcpkg install benchmark
```

Comment:
These expect that `vcpkg` is located in current directory, and only differ in slashes.
Can this be condensed into a single entry?

Reply:
will fix it